modules/SofaBoundaryCondition/src/SofaBoundaryCondition/FixedRotationConstraint.inl

```diff
@@ -87,9 +87,9 @@ void FixedRotationConstraint<DataTypes>::projectPosition(const core::MechanicalP
     for (unsigned int i = 0; i < x.size(); ++i)
     {
         // Current orientations
-        sofa::type::Quat<SReal> Q = x[i].getOrientation();
+        const sofa::type::Quat<SReal> Q = x[i].getOrientation();
         // Previous orientations
-        sofa::type::Quat<SReal> Q_prev = previousOrientation[i];
+        const sofa::type::Quat<SReal> Q_prev = previousOrientation[i];
```

Comment:
```suggestion
        const sofa::type::Quat<SReal>& Q_prev = previousOrientation[i];
```
same (could not do a multi-line suggestion 😕)

Reply:
Actually, you can select multiple lines before making the suggestion